app/models/ems_cluster.rb

```diff
@@ -230,6 +232,12 @@ def memory_reserve
     get_reserve(:memory_reserve)
   end
 
+  def lans
```

Comment:
should we move this to `AggregationMixin` together with the existing `all_storages`?
Possibly add a spec for it?